backtester/eventhandlers/portfolio/size/size.go

```diff
@@ -94,14 +96,17 @@ func (s *Size) calculateBuySize(price, availableFunds, feeRate float64, minMaxSe
 // eg BTC-USD baseAmount will be BTC to be sold
 // As fee calculation occurs during the actual ordering process
 // this can only attempt to factor the potential fee to remain under the max rules
-func (s *Size) calculateSellSize(price, baseAmount, feeRate float64, minMaxSettings config.MinMax) (float64, error) {
+func (s *Size) calculateSellSize(price, baseAmount, feeRate, sellLimit float64, minMaxSettings config.MinMax) (float64, error) {
 	if baseAmount <= 0 {
 		return 0, errNoFunds
 	}
 	if price == 0 {
 		return 0, nil
 	}
 	amount := baseAmount * (1 - feeRate)
+	if sellLimit > minMaxSettings.MinimumSize && sellLimit <= minMaxSettings.MaximumSize && sellLimit <= amount {
```

Comment:
This looks like it should be
```
if sellLimit >= minMaxSettings.MinimumSize && sellLimit <= minMaxSettings.MaximumSize && sellLimit <= amount {
```
I see no reason why the sell limit can't be used if its the same as the minimum

Reply:
@gloriousCode Done.